.github/workflows/reusable-unit-tests.yml

```diff
@@ -0,0 +1,43 @@
+name: "Unit tests"
+
+on:
+  workflow_call:
+    secrets:
+      CODECOV_TOKEN:
+        required: true
```

Comment:
Interesting that reviewdog is not complaining about a missing `description` here.

Reply:
It is weird, I've added it in anyway.